gramps_webapi/api/__init__.py

```diff
@@ -68,7 +68,8 @@ def register_endpt(resource: Type[Resource], url: str, name: str):
 register_endpt(BookmarkResource, "/bookmarks/<string:namespace>", "bookmark")
 register_endpt(BookmarksResource, "/bookmarks/", "bookmarks")
 # Filter
-register_endpt(FilterResource, "/filters/<string:namespace>", "filter")
+register_endpt(FilterResource, "/filters/<string:namespace>/<string:name>", "filter")
+register_endpt(FilterResource, "/filters/<string:namespace>", "filters")
```

Comment:
So now you have multiple endpoints for the same resource class and the different verbs will have different number of arguments. Does that work? What if I send a `GET` request to `/filters/namespace`? Won't that cause a 500 server error? Don't you have do define `name` as a keyword argument (with default value) on all the verbs' methods? And return 405 if it's set on the wrong method?

Reply:
Oh yeah, you're right, I'm not handling that right. I will fix that up, thanks!

Reply:
I think that is a bit better now. Boy I miss the obvious sometimes.